appserver/admingui/devtests/run_test.sh

```diff
@@ -87,6 +92,12 @@ run_test_id(){
   test_run
   merge_junit_xmls $WORKSPACE/main/appserver/admingui/devtests/target/surefire-reports
   change_junit_report_class_names
+}
+
+post_test_run(){
+  cp $WORKSPACE/bundles/version-info.txt $WORKSPACE/results/ || true
```

Comment:
Why this duplication? They are already copied in 75-77

Reply:
You can remove the lines 75-77

Reply:
Sure. will do.

Reply:
Done